_pages/index.md

```diff
@@ -17,22 +17,24 @@ permalink: /
     <div class="clearfix mxn2">
       <div class="col sm-col-4 px2 mb3">
         <img alt="" src="{{ site.baseurl }}/assets/img/users.svg" height="90">
-        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue">Simple and secure for our users</h2>
-        <p>Through login.gov, users can sign in to multiple government agencies. The goal: make managing benefits, services, and applications easier, faster, and more secure.</p>
-        <p>The system replicates the design, performance, and experience people are used to having when logging into their private email or any top tech industry website.</p> 
+        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue">Simple *and* secure for the public</h2>
+        <p>Because login.gov is a shared service,</p>
+        <p>* Users need fewer passwords and learn fewer interfaces
+        * Security experts protect one service instead of many.
+        * Dedicated teams of design and security experts will  continuously improve it.</p>
       </div>
       <div class="col sm-col-4 px2 mb3">
         <img alt="" src="{{ site.baseurl }}/assets/img/partners.svg" height="90">
-        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue">Partnering across government</h2>
-        <p>login.gov handles software development and customer support of a login system so agencies can focus on providing high-quality services for people to use after they sign in.</p>
+        <h2 class="mt2 mb2 pb2 blue border-bottom border-light-blue">Saving the federal government time and money</h2>
+        <p>llogin.gov handles software development, security operations, and customer support. This frees up government departments to focus on their missions while reducing costs and improving security.</p>
```

Comment:
Thanks! Hey, also, was trying to create bullets in GH and wasn't sure if I used to correct markdown, @hursey013.